features/step_definitions/analytics_steps.rb

```diff
@@ -1,20 +1,19 @@
 
 Then(/^the ecommerce tracking tags are present$/) do
   assert page.has_selector?("div#root[data-analytics-ecommerce]")
+  assert page.has_selector?('div#root[data-search-query]')
   assert page.has_selector?('div#root[data-ecommerce-start-index="1"]')
-  assert page.has_selector?('div#root[data-ecommerce-start-index="1"]')
-  assert page.has_selector?('div#root[data-list-title="Mainstream browse"]')
-  assert page.has_selector?('div#root[data-search-query="all categories"]')
+  assert page.has_selector?('div#root[data-list-title="Mainstream browse - Column 1"]')
 
-  assert_equal page.find("div#root li:nth-child(1) a")['data-ecommerce-row'], '1'
+  assert page.find("div#root li:nth-child(1) a")['data-ecommerce-row']
   assert_equal page.find("div#root li:nth-child(1) a")['data-ecommerce-path'], '/browse/benefits'
 
-  assert_equal page.find("div#root li:nth-child(2) a")['data-ecommerce-row'], '2'
+  assert page.find("div#root li:nth-child(2) a")['data-ecommerce-row']
```

Comment:
For consistency could this be?
```suggestion
  assert page.has_selector?("div#root li:nth-child(2) a['data-ecommerce-row']")
```

Reply:
Great shout, amended